tbtc-ts/src/bitcoin.ts

```diff
@@ -83,6 +90,74 @@ export type UnspentTransactionOutput = TransactionOutpoint & {
   value: number
 }
 
+/**
+ * Information about important parts of a transaction, including inputs and
+ * outputs.
+ */
+export interface Info {
+  /**
+   * Transaction's version.
+   */
+  txVersion: string
+
+  /**
+   * All transaction's inputs, prepended by the number of transaction inputs.
+   */
+  txInputVector: string
+
+  /**
+   * All transaction's outputs prepended by the number of transaction outputs.
+   */
+  txOutputVector: string
+
+  /**
+   * Transaction's locktime.
+   */
+  txLocktime: string
+}
```

Comment:
What do you think about naming it `DecomposedRawTransaction` and use just `version, inputs, outputs, locktime` as field names?

Reply:
Done.